### PR TITLE
Add Hex-String solutions to examples

### PR DESCRIPTION
**Pull request by Laurel-rao**
Signed-off-by: raojiajun <13694846652@163.com>

### CODE REVIEW

File: examples/Hex-String/main.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package main
```
Comment (TimRots) [inline examples/Hex-String/main.go:1]: Please add copyright/license header as with other examples
```
// Copyright 2012 The GoSNMP Authors. All rights reserved.  Use of this
// source code is governed by a BSD-style license that can be found in the
// LICENSE file.
```

File: examples/Hex-String/main.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+	for i, variable := range result.Variables {
+		fmt.Printf("%d: oid: %s ", i, variable.Name)
+
+		switch variable.Type {
+		case g.OctetString:
+			value := variable.Value.([]byte)
+			if strings.Contains(strconv.Quote(string(value)), "\\x") {
+				tmp := ""
+				for i := 0;i < len(value);i ++ {
```
Comment (TimRots) [inline examples/Hex-String/main.go:36]:
```go
				for i := 0; i < len(value); i++ {
```

File: examples/Hex-String/main.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+					tmp += fmt.Sprintf("%v", value[i])
+					if i != (len(value) - 1) {
+						tmp += " "
+					}
+				}
+				fmt.Printf("Hex-String: %s\n", tmp)
+			}	else {
```
Comment (TimRots) [inline examples/Hex-String/main.go:43]:
```go
} else {
```

**Review by TimRots — CHANGES_REQUESTED** [review]
Left some comments. 
You could run ``gofmt`` on the code to catch most of these issues.

**Comment by Laurel-rao** [comment]
i already adjust the style problem that we talk about it, please check code then merge the pull request

**Review by TimRots — APPROVED** [review]

**Pull request merged**